resources/views/components/language-changer.blade.php

```diff
@@ -3,26 +3,31 @@
 ])
 
 @if (count($model->languages) > 1)
-    <x-expander level="2" :summary="__('Change page language')">
-        <h3>{{ __('Available languages') }}</h3>
-        <ul role="list">
-            @foreach ($model->languages as $code)
-                @if (in_array($code, config('locales.supported')))
-                    {{-- Make sure at least the model name is translated to avoid 404 errors. --}}
-                    @if (!$model->isTranslatableAttribute($modelName) || !empty($model->getTranslation($modelName, $code, false)))
+    <div class="flex justify-between">
```

Comment:
Mobile layout looks a bit weird:

<img width="414" alt="Screen Shot 2022-10-21 at 9 41 12 AM" src="https://user-images.githubusercontent.com/605361/197198106-d86db844-fe67-4786-a428-f79213765700.png">

I'd suggest changing this container to flex-direction column and switching it to row at the medium breakpoint:

```suggestion
    <div class="flex flex-col gap-6 justify-between md:flex-row">
```

I'd also remove the `whitespace-nowrap` from the "Please contact support" paragraph as it causes it to overflow on mobile (note, this screenshot includes the suggested change above):

<img width="338" alt="Screen Shot 2022-10-21 at 9 44 43 AM" src="https://user-images.githubusercontent.com/605361/197198705-0efd569c-513e-45da-bc60-a63ccfff0ba8.png">